Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/risklevel/ui/TestRiskLevelCalculationFragmentCWAViewModel.kt

```diff
@@ -77,11 +78,11 @@ class TestRiskLevelCalculationFragmentCWAViewModel @AssistedInject constructor(
 
     fun retrieveDiagnosisKeys() {
         viewModelScope.launch {
-            try {
-                RetrieveDiagnosisKeysTransaction.start()
+            withContext(Dispatchers.IO) {
```

Comment:
Could use `CWAViewModel.launch`, which will use the viewmodel scope automatically + Dispatchers.Default.